histFactory/condorTools.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, sampleCfg, execPath, plotConfig, baseDir = ".", rescale = Fal
                 # NB : the returned file list does not contain exactly nEvt_tot*fraction event
                 #      it is rounded up to nEvt_tot * fraction + nEvt in next file
                 nEvt_tot = dbSample.nevents
-                nEvt_wanted = int(nEvt_tot * sample["sample_fraction"]) + (nEvt_tot % sample["sample_fraction"] > 0)
+                nEvt_wanted = int(nEvt_tot * sample["sample_fraction"]) + (nEvt_tot * sample["sample_fraction"] % 1 != 0)
```

Comment:
I'd prefer to import math ;) This is rather cryptic, while `math.ceil` is understood universally :smile: